### PR TITLE
include all icons in breadcrumbs dropdown

### PR DESCRIPTION
**Pull request by songgao**
The icon for `/keybase` isn't right yet; pending on a comment on Zeplin.

Screenshot:

screen shot 2018-04-05 at 7 32 38 pm

### CODE REVIEW

File: shared/fs/header/container.js [hunk 1]
```diff
@@ -73,18 +68,17 @@ const mergeProps = (
     }
   })
   let breadcrumbItems = items || []
-  let dropdownItems = []
+  let dropdownPath = Types.stringToPath('')
```
[the remark this answers is not in the document]
Comment (songgao) [reply shared/fs/header/container.js]: Good catch; will fix thanks!

**Comment by songgao** [comment]
> clicking the dropdown doesn't work because it's not in the routeDef

@buoyad Thanks for noticing that! I think KBFS-2850 is specifically for fixing that.

File: shared/fs/header/container.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (buoyad) [reply shared/fs/header/container.js]: sounds good!

**Comment by buoyad** [comment]
@songgao ah I see, I forgot to check the kbfs board. Cool!

**Pull request merged**